check_py_coverage.sh

```diff
@@ -34,7 +34,8 @@ else
     fi
 
     cd $BUILD_DIR
-    $DOWNLOADER https://pypi.python.org/packages/source/c/coverage/coverage-3.7.1.tar.gz #md5=67d4e393f4c6a5ffc18605409d2aa1ac
+    #SHA256=56e448f051a201c5ebbaa86a5efd0ca90d327204d8b059ab25ad0f35fbfd79f1
```

Comment:
Do you need this commented line?

Reply:
Yes...it is the SHA256 code (It is needed for tracking purposes)